hbase-it/src/test/java/org/apache/hadoop/hbase/IntegrationTestsDriver.java

```diff
@@ -77,7 +77,7 @@ protected void addOptions() {
 
   @Override
   protected void processOptions(CommandLine cmd) {
-    String testFilterString = cmd.getOptionValue(SHORT_REGEX_ARG, null);
+    String testFilterString = cmd.getOptionValue(SHORT_REGEX_ARG, (String) null);
```

Comment:
We can simply skip/remove 2nd param as the method anyways return null by default.